staging/src/k8s.io/component-base/config/v1alpha1/types.go

```diff
@@ -141,3 +141,18 @@ type VModuleItem struct {
 	// that match the pattern.
 	Verbosity uint32 `json:"verbosity"`
 }
+
+// TracingConfiguration provides versioned configuration for tracing clients.
+type TracingConfiguration struct {
+	// +optional
+	// Endpoint of the collector this component will report traces to.
+	// The syntax is defined in https://github.com/grpc/grpc/blob/master/doc/naming.md.
+	// Default is the otlp grpc default, localhost:4317
+	// The connection is insecure, and does not currently support TLS.
+	Endpoint *string `json:"endpoint,omitempty"`
+
+	// +optional
+	// SamplingRatePerMillion is the number of samples to collect per million spans.
+	// Defaults to 0.
```

Comment:
See comments on the internal type above.

Reply:
removed the `Default` line and modified